<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="loger" name="cpu_loger" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cpu_max_persent">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>- 100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpu_min_persent">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>75</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- 0%</string>
    </property>
   </widget>
   <widget class="loger" name="memory_loger" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="memory_max_persent">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="memory_min_persent">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>75</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0GB</string>
    </property>
   </widget>
   <widget class="QLabel" name="memory_now">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cpu_core">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>loger</class>
   <extends>QWidget</extends>
   <header>loger.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
